chapter-01/recipe-09/README.md

```diff
@@ -1,6 +1,8 @@
 # Setting the standard for the language
 
-Abstract to be written ...
+We will show various mechanism to require and set a specific compiler standard
```

Comment:
I think the title is better. I would have said something like:

We will show various mechanism to require and set a specific language standard in a compiler-agnostic way for our project.